model/serializer.go

```diff
@@ -86,34 +86,35 @@ func (r jsonSerializer) jMap(in any) (out any) {
 	}
 	switch t.Kind() {
 	case reflect.Struct:
-		mp := make(map[string]any)
+		out = reflect.New(t).Interface()
```

Comment:
**hint**: Need to build a new struct (instead of map) to preserve field tags for json serializing.